components/layout/Header.jsx

```diff
@@ -0,0 +1,24 @@
+import { Span } from "../ui/Typography";
+import Anchor from "../ui/Anchor";
+
+export default function Header() {
+  return (
+    <div className="grid grid-cols-1 sm:grid-cols-2 mb-13.25 sm:mb-44 xl:mb-64.5">
+      <div className=" sm:col-start-2 flex flex-wrap sm:flex-nowrap">
+        <div className="flex-1 sm:flex-none sm:w-auto xl:w-1/2 sm:border-l-0.5 border-black pt-5 pr-4 sm:pr-2.5 pb-4.25 sm:pl-2.5 sm:pt-10 sm:pb-0.5 xl:pl-5 xl:pb-0">
```

Comment:
rather than using the flex-none which stops an element from shrinking, you can use flex-initial to override the flex-1 property